docs/API.md

```diff
@@ -26,6 +27,7 @@
 | `registerLanguage` | `function()` | Registers a custom Language within the editor
 | `editorStyle` | `string` | Additional Styling applied to Editor Container
 | `theme` | `string` | Theme used to style the Editor
-| `isElectronApp` | `function()` | Returns true or false based on if running in Electron.
+| `isElectronApp` | `function()` | Returns true or false based on if running in Electron
+| `[(ngModel)]` | `directive` | One-way or two-way binding of the variable value provided
```

Comment:
Typically there is no need to add this as a property, we can just state that we have forms support.